### PR TITLE
Adding Guillaume Allain as Bagger

### PR DESCRIPTION
**Pull request by gallain**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -13207,6 +13207,80 @@ var data = {
...
+            "fr"
+          ]
+        }
+      ],
+      "cities": [
+        "Nantes"
+      ],
+      "contacts": {
+        "twitter": "gallain",
+        "mail": "guillaume@llain.fr"
```
Comment (dadoonet) [inline baggers.js:13282]: Just checking. Is your email address correct? I guess so but prefer double checking

File: baggers.js [hunk 2]
```diff
@@ -13207,6 +13207,80 @@ var data = {
...
+    },
+    {
+      "since": "2016-07-18",
+      "name": "Guillaume Allain",
+      "bio": "Associé chez Lucca, je suis product owner chez Lucca  depuis 2007. J'ai pu vivre le passage d'un éditeur Saas de 4 à 50 collaborateurs en 9 ans et participer à la structuration de l'organisation à chaque étape de la croissance.",
+      "picture": "https://secure.gravatar.com/avatar/4bd61d872d5e2596c631a4f1d3041451",
+      "websites": [
+        {
+          "name": "Lucca",
+          "url": "https://http://www.lucca.fr/"
```
Comment (Nilhcem) [inline baggers.js]: `"url": "https://www.lucca.fr"`

**Comment by dadoonet** [comment]
@gallain Could you fix the issue please so we can merge it?

File: baggers.js (hunk 1 above)
Comment (gallain) [inline baggers.js:13282]: Yes it is

**Pull request merged**